config/locales/hyrax.en.yml

```diff
@@ -63,16 +63,16 @@ en:
         rights_statement: Rights
         description:      Description
   hyrax:
-    product_name:           "Hyrax"
-    product_twitter_handle: "@SamveraRepo"
-    institution_name:       "Institution"
-    institution_name_full:  "The Institution Name"
-    account_name:           "My Institution Account Id"
+    product_name:           "OHSU Scholar Archive"
+    product_twitter_handle:
+    institution_name:       "OHSU"
+    institution_name_full:  "OHSU"
+    account_name:
     directory:
-      suffix:               "@example.org"
+      suffix:
     footer:
-      copyright_html: "<strong>Copyright &copy; 2017 Samvera</strong> Licensed under the Apache License, Version 2.0"
-      service_html: A service of <a href="http://samvera.org/" class="navbar-link" target="_blank">Samvera</a>.
+      copyright_html:
```

Comment:
Copyright should probably be `Copyright &copy; 2018 OHSU`

And we could leave the Samvera service link in.